src/chains.jl

```diff
@@ -26,9 +26,10 @@ end
 
 # Generic chain constructor.
 function Chains(
-    val::AbstractArray{<:Union{Missing, Real},3},
+    val::AbstractArray{<:Union{Missing,Real},3},
     parameter_names::AbstractVector{Symbol} = Symbol.(:param_, 1:size(val, 2)),
     name_map = (parameters = parameter_names,);
+    weights=StatsBase.UnitWeights{Bool}(size(val, 1)),  # TODO: Multiple weighted chains
```

Comment:
This comment is quite confusing IMO. I suggest removing it and opening a more concrete issue instead since nobody will browse the code for TODOs:
```suggestion
    weights=StatsBase.UnitWeights{Bool}(size(val, 1)),
```
Alternatively, clarify it and write e.g.
```suggestion
    weights=StatsBase.UnitWeights{Bool}(size(val, 1)),  # TODO: Different weights per chain (StatsBase only supports weight vectors currently)